### ISSUE TITLE
Missing image LiquidOre.png

### ISSUE THREAD

**Issue by chaos234** [aidancbrady/Mekanism#3491]
Since you have switched the images into the subdir liquids there is an error with the image "LiquidOre.png" which isn't found.

This results that some liquid tankes are nice black-pink colored because the texture is missing.

**Comment by aidancbrady** [comment]
Not experiencing this. Make sure your assets are totally up to date, I remember I fixed this a while back.

**Comment by chaos234** [comment]
But I do this @aidancbrady .

I have fresh forked your repro and seted it up correctly on my PC.
Then created a new branch called "opaque" from the current development branch wich is for 1.7.10.

After adding all the stuff correctly and run gradle build all went fine. Then I putted this new JAR in my mods folder an then getting the message, that the image is not found.

Which place in the code is it defined? Because I also checked all double befor I have implemented my stuff.
